index.html

```diff
@@ -1,102 +1,56 @@
+      <p class="c-site-header__subtitle">(Upcoming Software Developer)</p>
     </header>
-    <main tabindex="0">
-      <section id="about">
-        <header><h2>About Me</h2></header>
-        <p>
-          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Nulla nisi
-          excepturi quidem, eum rem doloribus quam aspernatur hic enim eligendi
-          commodi minima ullam necessitatibus, cumque blanditiis, nihil magni
-          amet consectetur?
-        </p>
-      </section>
-      <section id="projects">
-        <header><h2>Projects Showcase</h2></header>
-        <p>
-          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Nulla nisi
-          excepturi quidem, eum rem doloribus quam aspernatur hic enim eligendi
-          commodi minima ullam necessitatibus, cumque blanditiis, nihil magni
-          amet consectetur?
-        </p>
-      </section>
-      <section id="contact">
-        <header><h2>Contact me</h2></header>
-        <p>
-          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Nulla nisi
-          excepturi quidem, eum rem doloribus quam aspernatur hic enim eligendi
-          commodi minima ullam necessitatibus, cumque blanditiis, nihil magni
-          amet consectetur?
+
+    <nav aria-label="Main Site Links" class="c-site-nav">
+      <a href="index.html" class="c-site-nav__link">Home</a>
+      <a href="about.html" class="c-site-nav__link">About</a>
+      <a href="contact.html" class="c-site-nav__link">Contact</a>
+    </nav>
+
+    <main>
+      <section>
+        <img src="m pic.png" alt="Fathi Kahin" class="c-section__image" />
+        <h2 class="c-section__title">About Me</h2>
+        <p class="c-section__text">
+          I am a biomedical scientist turned refugee case worker at a charity
+          supporting refugees and asylum seekers in their integration into
+          society. My daily tasks involve aiding clients in finding suitable
+          educational facilities and housing, as well as providing independent
+          information and guidance. My role is multi-faceted, with a focus on
+          facilitating access to welfare support and financial benefits,
+          especially for those unfamiliar with the system. In addition to my
+          work, I am passionate about technology and its potential to transform
+          lives. I discovered Code Your Future while researching services for
+          clients and found it perfectly aligned with my values. Assisting
+          refugees in rebuilding their lives has always been a personal goal,
+          and coding is my dream avenue for achieving this. I'm eager to learn
+          programming and apply my skills to make a positive impact. Committed
+          to continuous learning, I plan to leverage my experience as a refugee
+          case worker to excel in the tech sector. <br /><br />
         </p>
+        <h2 class="c-section__title">My Interests</h2>
+        <ul class="c-section__list">
```

Comment:
make sure to add links to your existing projects! I cannot see any of your past work on this page!